src/modules/core/components/InfoPopover/MemberInfoPopover/UserReputation.tsx

```diff
@@ -66,51 +69,64 @@ const UserReputation = ({
         }}
         text={MSG.labelText}
       />
-      {isEmpty(formattedUserReputations) ? (
-        <p className={styles.noReputationDescription}>
-          <FormattedMessage
-            {...MSG.noReputationDescription}
-            values={{ isCurrentUserReputation }}
-          />
-        </p>
+      {isUserReputationLoading ? (
+        <SpinnerLoader
+          appearance={{
+            theme: 'grey',
+            size: 'small',
+          }}
+        />
```

Comment:
How did this changes end up in here ? These are not related to your branch.

Did you cherry-pick them from somewhere? If so, why ?

Reply:
It looks like there are these commits from @arrenv earlier today that ended up in my branch, https://github.com/JoinColony/colonyDapp/pull/3528/commits/0f5a455395c052840ab39ab9752e75103486313f
I think this is the cause of these changes.

Reply:
Yeah, some rebase / merge screwed up your branch. We'll fix it tommorow.